src/liboslexec/llvm_ops.cpp

```diff
@@ -91,7 +91,15 @@ examples), as you are just coding in C++, but there are some rules:
 */
 
 
+// Some gcc versions on some platforms seem to have max_align_t missing from
```

Comment:
We don't use it in our code. It's referenced in some standard headers. The conflict comes from the setup on Travis (and I assume this can be the case for external users as well) compiling  with llvm_ops.cpp with clang 3.4 but having gcc6 as the main system compiler. I think some of the logic clang uses to figure out which headers it can get from the system compiler and which it supplies itself ends up somehow getting a combination in which this definition is referenced but not provided. I'm assuming it will be resolved when we upgrade to newer clang/llvm versions, which I'm actively working on (between fires).